chakra-ui-steps/src/components/VerticalStep/index.tsx

```diff
@@ -99,7 +99,10 @@ export const VerticalStep = React.forwardRef<HTMLDivElement, VerticalStepProps>(
         </Flex>
         <chakra.div
           className="cui-steps__vertical-step-content"
-          __css={{ minH: '8px', ps: `calc(${stepIconContainer.width})` }}
+          __css={{
+            minH: '8px',
+            paddingLeft: stepIconContainer?.width ? `calc(${stepIconContainer.width}` : 0,
```

Comment:
I think we should use `paddingStart` here so that this value respects RTL layouts also